src/tests/ftest/container/attribute.py

```diff
@@ -206,15 +249,24 @@ def test_container_attribute_asyn(self):
                 self.fail("RC not as expected after set_attr First {0}"
                           .format(GLOB_RC))
 
-            # skip-step due to DAOS-6880
-            #GLOB_SIGNAL = threading.Event()
-            #size, buf = self.container.container.list_attr(cb_func=cb_func)
-            #GLOB_SIGNAL.wait()
-            #if GLOB_RC != 0 and expected_result in ['PASS']:
-            #    self.fail("RC not as expected after list_attr First {0}"
-            #              .format(GLOB_RC))
-            #if expected_result in ['PASS']:
-            #    self.verify_list_attr(attr_dict, size, buf, mode="async")
+            # Workaround
+            # Due to DAOS-7093 skip the usage of pydaos cont list attr
+            # GLOB_SIGNAL = threading.Event()
+            #
+            # size, buf = self.container.container.list_attr(cb_func=cb_func)
+            #
+            out_attr_dict = self.daos_cmd.container_list_attrs(
+                pool=self.pool.uuid,
+                cont=self.container.uuid)
+
+            # GLOB_SIGNAL.wait()
+            # if GLOB_RC != 0 and expected_result in ['PASS']:
+            #     self.fail("RC not as expected after list_attr First {0}"
+            #               .format(GLOB_RC))
```

Comment:
Why these comments still in code?

Reply:
When 7085 is resolved, this lines can be re-added

Reply:
Aren't we supposed to use skipForTicket pragma for this issue?